src/Shared/Recruit.Vacancies.Client/Ioc/ServiceCollectionExtensions.cs

```diff
@@ -61,7 +61,7 @@ public static void AddRecruitStorageClient(this IServiceCollection services, ICo
                 .Configure<AccountApiConfiguration>(configuration.GetSection("AccountApiConfiguration"))
```

Comment:
Change the namespace to point to the correct location.

Reply:
Remove unused namespace please.

Reply:
👍